package.json

```diff
@@ -33,7 +33,7 @@
     "eslint-config-airbnb": "15.1.0",
     "eslint-plugin-import": "2.12.0",
     "eslint-plugin-jsx-a11y": "5.1.1",
-    "eslint-plugin-react": "7.5.1",
+    "eslint-plugin-react": "7.10.0",
```

Comment:
This looks safe, since the new version of the plugin still supports ESLint 3:
https://github.com/yannickcr/eslint-plugin-react/blob/157cc932be2cfaa56b3f5b45df6f6d4322a2f660/package.json#L39-L41